src/app/modules/project/overview/overview.component.html

```diff
@@ -99,67 +119,53 @@ <h3>Filter</h3>
-      <button><img alt="" src="assets/images/research.svg" loading="lazy"></button>
+      <input [formControl]="searchControl" placeholder="search project" type="search">
+      <button><img alt="" loading="lazy" src="assets/images/research.svg"></button>
     </div>
 
     <ng-container *ngIf="!projectsEmpty() && !projectsLoading; else noProjects">
-      <table class="projects-table" role="presentation">
-        <tr *ngIf="projectsLoading">
-          <td colspan="3">Loading...</td>
-        </tr>
-        <tr *ngFor="let project of projectsToDisplay">
-          <td class="title" (click)="onClickProject(project.id, project.name)">
-            <div class="icon-placeholder">
-              <img align="middle" alt="project-image" [src]="getIconUrl(project.id)" onerror="this.src='/assets/images/placeholder.svg';" loading="lazy">
-            </div>
-            <span>
-              <h3>{{ project.name }}</h3>
-              <p>{{ project.shortDescription }}</p>
-            </span>
-          </td>
-          <td class="icon d-none d-xl-table-cell">
-            <p class="large"><img alt="clock" class="icon"
-                src="assets/images/clock.svg" loading="lazy">{{ project.created | date:'longDate' }}</p>
-          </td>
-          <td class="icon d-none d-xl-table-cell">
-            <p class="large"><img alt="user" class="icon"
-                src="assets/images/user.svg" loading="lazy">{{ project.user.name }}</p>
-          </td>
-        </tr>
-      </table>
+      <div class="project-list-wrapper" [ngClass]="this.showListView ? 'list' : 'grid'">
+        <h2 *ngIf="projectsLoading">Projects loading...</h2>
+        <app-project
+            *ngFor="let project of projectsToDisplay"
+            [project]="project"
+            [showListView]="this.showListView"
+            (click)="onClickProject(project.id, project.name)">
+        </app-project>
+      </div>
     </ng-container>
+
     <ng-template #noProjects>There are no projects available.</ng-template>
+
     <div *ngIf="amountOfProjectsOnSinglePage < totalNrOfProjects && showPaginationFooter" class="pagination-div">
-      <pagination [(totalItems)]="totalNrOfProjects" [(itemsPerPage)]="amountOfProjectsOnSinglePage"
-        [customNextTemplate]="nextTemplate" [customPreviousTemplate]="prevTemplate" (pageChanged)="pageChanged($event)" class="pagination-footer">
+      <pagination (pageChanged)="pageChanged($event)" [(itemsPerPage)]="amountOfProjectsOnSinglePage"
+                  [(totalItems)]="totalNrOfProjects" [customNextTemplate]="nextTemplate" [customPreviousTemplate]="prevTemplate" class="pagination-footer">
       </pagination>
-      <ng-template #nextTemplate [customPreviousTemplate]="prevTemplate" let-disabled="disabled" let-currentPage="currentPage">
+      <ng-template #nextTemplate [customPreviousTemplate]="prevTemplate" let-currentPage="currentPage" let-disabled="disabled">
```

Comment:
I get the following warning in my console: "Can't bind to 'customPreviousTemplate' since it isn't a known property of 'ng-template'." when visiting the project overview page.